SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -801,4 +807,109 @@ void FLocalDeploymentManager::SetAutoDeploy(bool bInAutoDeploy)
 	bAutoDeploy = bInAutoDeploy;
 }
 
+void SPATIALGDKSERVICES_API FLocalDeploymentManager::TakeSnapshot(UWorld* World, bool bUseStandard,
+																  FSpatialSnapshotTakenFunc OnSnapshotTaken)
+{
+	FHttpModule& HttpModule = FModuleManager::LoadModuleChecked<FHttpModule>("HTTP");
+	TSharedRef<IHttpRequest> HttpRequest = HttpModule.Get().CreateRequest();
+
+	HttpRequest->OnProcessRequestComplete().BindLambda(
+		[World, bUseStandard, OnSnapshotTaken](FHttpRequestPtr HttpRequest, FHttpResponsePtr HttpResponse, bool bSucceeded) {
+			if (!bSucceeded)
+			{
+				UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Failed to trigger snapshot at '%s'; received '%s'"),
+					   *HttpRequest->GetURL(), *HttpResponse->GetContentAsString());
+				if (OnSnapshotTaken != nullptr)
+				{
+					OnSnapshotTaken(false, FString());
+				}
+				return;
+			}
+
+			// Unfortunately by the time this callback happens, the files haven't been flushed, so if you copy you may get
+			// the wrong info! So let's wait a bit..
+			FTimerHandle TimerHandle;
+			World->GetTimerManager().SetTimer(
+				TimerHandle,
+				[bUseStandard, OnSnapshotTaken]() {
+					bool bSuccess = false;
+
+					FString NewestSnapshotFilePath;
+					FString SnapshotSearchDirectory;
+
+					if (bUseStandard)
```

Comment:
Can we just get a comment explaining that the Standard Runtime places snapshots in logs and the CompatibilityMode sticks them in AppData